boutiques_descriptors/micapipe_0_2_2.json

```diff
@@ -0,0 +1,692 @@
+{
+  "name":           "micapipe",
+  "tool-version":   "0.2.2",
+  "container-image": {
+    "image": "micalab/micapipe:v0.2.2",
+    "index": "docker://",
+    "type": "singularity"
+  },
+  "schema-version": "0.5",
+  "author":         "Multimodal Imaging and Connectome Analysis (MICA) Lab ",
+  "url":            "https://github.com/MICA-MNI/micapipe",
+  "description":    "Please cite: Cruces RR, Royer J, Herholz P, Lariviere S, Vos de Wael R, Paquola C, Benkarim O, Park BY, Degre-Pelletier J, Nelson MC, DeKraker J, Leppert IR, Tardif C, Poline JB, Concha L, Bernhardt BC. Micapipe: A pipeline for multimodal neuroimaging and connectome analysis. Neuroimage. 2022 Nov;263:119612. doi: 10.1016/j.neuroimage.2022.119612. Epub 2022 Sep 5. PMID: 36070839. doi: 10.1016/j.neuroimage.2022.119612",
+  "command-line": "micapipe [BIDS_DATASET] [OUTPUT_DIR] [SUB] [SES] [FS_LICENCE] [PROC_STRUCTURAL] [PROC_SURF] [POST_STRUCTURAL] [PROC_FLAIR] [PROC_FUNC] [PROC_DWI] [SC] [MPC] [GD] [QC] [QC_SUBJ] [CLEANUP] [ATLAS] [FREESURFER] [T1WSTR] [UNI] [MF] [SURF_DIR] [T1] [REGSYNTH] [TRACTS] [KEEP_TCK] [AUTOTRACT] [THREADS] [TMPDIR] [ACQSTR] [DWI_MAIN] [DWI_RPE] [DWI_PROC] [RPE_ALL] [REGAFFINE] [DWIACQ] [BTHR] [NO_BVAL] [WSC] [TCK] [FLAIRSTR] [MPC_ACQ] [MPC_IMG] [MPC_REG] [FUNC_MAIN] [FUNC_PE] [FUNC_RPE] [TOPUP] [SESANAT] [FIX] [NOFIX] [NSR] [GSR] [DROPTR] [NOFC]",
+  "inputs": [
+      {
+        "description": "Path to the BIDS Dataset directory",
+        "id": "bids_dataset",
+        "name": "BIDS directory",
+        "optional": false,
+        "type": "File",
+        "value-key": "[BIDS_DATASET]",
+        "command-line-flag": "-bids"
+      },
+      {
+        "description": "Path to the output directory where derivatives will be stored",
+        "id": "output_dir",
+        "name": "Output directory (derivatives)",
+        "optional": false,
+        "type": "File",
```

Comment:
Remember to change the type of output_dir to 'String' and add the `mkdir -p` in the command.